src/blocks/modal.css

```diff
@@ -38,24 +38,22 @@
 }
 
 .modal__close {
-  background-image: url("../images/close-button.svg");
+  background-image: url(../images/close_icon.svg);
```

Comment:
Please, don't change your old correct styles of `.modal__close`. Your other modals have the same close icon and it's correct. You only need to change the class of the avatar close icon and nothing else. 

The project won't starts with errors https://disk.yandex.ru/i/pl2A-iquW_ELLw